dip-0003.md

```diff
@@ -127,8 +128,7 @@ The transaction consists of the following data in the payload area:
 | scriptPayout | Script | Variable | Payee script (p2pkh/p2sh) |
 | inputsHash | uint256 | 32 | The SHA256 hash of all the outpoints of the transaction inputs |
 | platformNodeID | byte[] | 0 or 20 | Dash Platform P2P node ID, derived from P2P public key. Only present for masternode type 1. |
-| platformP2PPort | uint_16 | 0 or 2 | TCP port of Dash Platform peer-to-peer communication between nodes (network byte order). Only present for masternode type 1. |
-| platformHTTPPort | uint_16 | 0 or 2 | TCP port of Platform HTTP/API interface (network byte order). Only present for masternode type 1. |
+| platformInfo | byte[] | variable | Platform network information (see [appendix](#appendix-c-network-information)) |
```

Comment:
I think it's better to rename `platformInfo` to  `platformNetAddress` or something similar; because `platformNodeId` is also kind of platform information, but it's not included here.

`platformNetInfo` ([see dip-0028 changes](https://github.com/dashpay/dips/pull/162/files#diff-e8ef02796b8d1585f1c72f16329ccb620cbb43d78d411c4348ecfded5ccc8b5bR240)) below is better name than `platformInfo`, but I think better to clarify it